axelor-crm/src/main/resources/i18n/messages_fr.csv

```diff
@@ -531,6 +536,7 @@
 "The end date must be after the start date","La date de fin doit être après la date de début",,
 "The number of repetitions must be greater than 0","Le nombre de répétitions doit être supérieur à 0",,
 "There's no configured sequence for opportunities for the company %s","Aucune séquence configurée pour les opportunités pour la société %s",,
+"These opportunities have been found for the same prospect, would you like to close it (closed losed) ?","Ces autres opportunités ont été trouvé pour le même prospect, souhaitez vous les fermer (fermée perdue) ?",,

```

Comment:
Please change the fr translation to "Ces autres opportunités ont été trouvé**es** pour le même prospect, souhaitez vous les fermer (fermée perdue) ?"